data/campaigns/Northern_Rebirth/scenarios/06a_Old_Friend.cfg

```diff
@@ -235,7 +235,7 @@
         [message]
             id=Rakshas
             # wmllint: local spelling Pah
-            message= _ "Pah! I would like to see those cowardly dwarves try their treachery on me as they did on the Black-Tusk! In any case, those dwarves will soon be groveling at my feet and equipping my people with the finest weapons. With the dwarvish weapons, and the wealth of Knalga, nothing will be able to stop us. Even that little backwater that you humans call Wesnoth will be nothing but a burnt out, pillaged wreck before long."
+            message= _ "Pah! I would like to see those cowardly dwarves try their treachery on me as they did on the Black-Tusk! In any case, those dwarves will soon be groveling at my feet and equipping my people with the finest weapons. With the dwarvish weapons, and the wealth of Knalga, nothing will be able to stop us. Even that little backwater that you humans call Wesnoth will be nothing but a burnt out, pillaged wreck before long!"
```

Comment:
Both seem similar to me, it works with a full-stop because Rakshas is just stating facts as he sees them. Thus my preference is for no change, rather than leaving the translators to find a single-character change in a long string.

Reply:
Okay we can leave this one as the original for the sake of translators please.